libs/features/mas/web-components/src/cdt/countdown-timer.js

```diff
@@ -0,0 +1,119 @@
+import { LitElement, html, css } from 'lit';
+import { styles } from './countdown-timer.css.js';
+
+export default class CountdownTimer extends LitElement {
+  static properties = {
+    label: { type: String },
+    daysHoursMins: { type: String },
+    timeRanges: { type: String },
+    daysLeft: { type: String },
+    hoursLeft: { type: String },
+    minutesLeft: { type: String },
+    isVisible: {type: Boolean},
+  };
+
+  #timeWords = [];
+  #timeRangesEpoch = [];
+
+  static styles = [styles];
+
+  constructor(endDateTime) {
+    super();
+    this.label = '';
+    this.daysHoursMins = '';
+    this.timeRanges = '';
+    this.daysLeft = '';
+    this.hoursLeft = '';
+    this.minutesLeft = '';
+    this.isVisible = false;
+  }
+
+  firstUpdated() {
+    this.#timeWords = this.daysHoursMins.split(' ');
+    this.#timeRangesEpoch = this.timeRanges.split(',');
+    this.countdownStart();
+  }
+
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    clearInterval(this.countdownInterval);
+  }
+
+  countdownCompleted() {
+    this.isVisible = false;
+    clearInterval(this.countdownInterval);
+  }
+
+  countdownStart() {
+    this.countdownUpdate();
+    this.countdownInterval = setInterval(() => {
+      this.countdownUpdate();
+    }, 60000);
```

Comment:
where does come the value of 60000? can you make it a constant and add a line of js doc if it is an important value?

Reply:
60000 is one minute in ms. Added a constant for representation